frontend/libs/ui-modules/src/modules/segments/utils/index.tsx

```
@@ -0,0 +1,159 @@
+}
+
+export const groupFieldsByType = (fields: any[]) => {
+  return fields.reduce((acc, field) => {
+    const value = field.value;
+    let key;
+
+    if (field.group) {
+      key = field.group;
+    } else {
+      key =
+        value && value.includes('.')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)